src/components/Header/Header.js

```diff
@@ -38,14 +38,15 @@ export default class Header extends Component {
         <Row className="navigation-container">
           <Col>
             <Navbar expand="lg">
-              <Navbar.Brand>MixNJuice</Navbar.Brand>
+              <Navbar.Brand className="pt-0">
+                <img src="/media/logo.svg" alt="logo" width="175" />
```

Comment:
Would you mind using SCSS to set the width?

Reply:
👍 Good idea, updated that!